tests/core/pyspec/eth2spec/test/altair/block_processing/test_process_sync_committee.py

```diff
@@ -284,6 +284,7 @@ def test_invalid_signature_previous_committee(spec, state):
 @with_all_phases_except([PHASE0, PHASE1])
 @spec_state_test
 @always_bls
+@with_configs([MINIMAL], reason="too slow")
```

Comment:
1 mainnet sync period is about a day right? 225 epochs per day, 256 per sync period. That will take a very long time to run in a python test indeed (even worse with BLS enabled in generators, if it involves any blocks)